.gitattributes

```diff
@@ -1,4 +1,3 @@
-dune             linguist-generated
```

Comment:
Isn't this file still generated?

Reply:
Ah you meant `/dune` (didn't notice there was one). I'll fix that.